src/ApiCore/GapicClientTrait.php

```diff
@@ -181,45 +226,42 @@ protected function setClientOptions(array $options)
             $clientConfig,
             $options['disableRetries']
         );
-        $gapicVersion = isset($options['gapicVersion'])
-            ? $options['gapicVersion']
-            : self::getGapicVersion($options);
         $this->agentHeaderDescriptor = new AgentHeaderDescriptor([
-            'libName' => $this->pluck('libName', $options, false),
-            'libVersion' => $this->pluck('libVersion', $options, false),
-            'gapicVersion' => $gapicVersion,
+            'libName' => $options['libName'],
+            'libVersion' => $options['libVersion'],
+            'gapicVersion' => $options['gapicVersion'] ?: self::getGapicVersion($options),
         ]);
 
         self::validateFileExists($options['descriptorsConfigPath']);
         $descriptors = require($options['descriptorsConfigPath']);
         $this->descriptors = $descriptors['interfaces'][$this->serviceName];
 
-        $authConfig = $options['authConfig'] ?: [];
-        $this->authWrapper = $this->createAuthWrapper($options['auth'], $authConfig);
+        $this->authWrapper = $this->createCredentialsWrapper($options['credentials'], $options['credentialsConfig']);
```

Comment:
Would $this->authWrapper need an update to $this->credentialsWrapper?

Reply:
It got this update in the PR merged today: https://github.com/googleapis/gax-php/pull/175/files#diff-c38cfca583ea39046b0c40fcce3dcbe9R246